CameraOverlay/overlay_all_stats.py

```diff
@@ -10,13 +10,14 @@ class OverlayAllStats(Overlay):
 			topics.DAS.stop,
 			topics.PowerModel.recommended_sp,
 			topics.PowerModel.max_speed,
-			topics.DAShboard.recieve_message,
+			topics.DAShboard.receive_message,
 	]
 
 	def __init__(self):
 		super(OverlayAllStats, self).__init__()
 		self.text_height = 50
 		self.speed_height = 70
+		self.frames_per_second = 14
```

Comment:
Frames per second shouldn't be fixed, it will be higher or lower based on how fast your computer is.